frontend/src/lib/getChartData.ts

```diff
@@ -17,46 +17,46 @@ const getChartData = ({ queryData }: GetChartDataProps): ChartData => {
 	const labels = Array.from(uniqueTimeLabels).sort((a, b) => a - b);
 
 	const response = queryData.map(
-		({ queryData, query: queryG, legend: legendG }) =>
-			queryData.map((e) => {
+		({ queryData: queryDataNested, query: queryG, legend: legendG }) =>
```

Comment:
```suggestion
		({ queryData, query: queryG, legend: legendG }) =>
```

rename above function param by allQueryData